### ISSUE TITLE
When the name of the test case contains Chinese, the output log shows garbled code. How to fix this？

### ISSUE THREAD

**Issue by xuegj1010** [robotframework/RIDE#2170]
The version of RIDE is 1.7.4.1
<img width="917" alt="1" src="https://user-images.githubusercontent.com/21315470/77537797-a5fcfb00-6ed9-11ea-8286-47c1a9f6ac17.png">

**Comment by HelioGuilherme66** [comment]
@xuegj1010 I see that `robot` is installed in Python 3.7, but is RIDE also installed in Python 3.7?
Anyway, if you can try to install the current development version of RIDE (see README.md), it may work fine. See here my screenshot of it installed in Python 3.8:
![image](https://user-images.githubusercontent.com/2866630/77586878-cd09fb80-6ede-11ea-8898-c22d7a377c96.png)

Please confirm, from Tools-:Preferences that the Font in Test Runner is **Courier New**.

**Comment by xuegj1010** [comment]
@HelioGuilherme66 Thank you very much! By referring to #2053, I have solved this problem, I did not upgrade my RIDE, I modified the source code, I look forward to your version 2.0!

**Comment by HelioGuilherme66** [comment]
@xuegj1010 Please confirm that you have RIDE installed in Python 2.7.

**Comment by gogily** [comment]
Hi modify your file"testrunnerplugin.py"
change “'SYSTEM': SYSTEM_ENCODING,” to "'SYSTEM': OUTPUT_ENCODING,"

**Comment by xuegj1010** [comment]
@gogily Thanks, it worked！
@HelioGuilherme66 My RIDE is installed in Python3.7.3

**Comment by HelioGuilherme66** [comment]
Well, on Windows 10 and with Python 3.8, latest development version I don't need that change. This is my simple test suite, in path "D:\Testes\无纸化\teste2.robot":
```robot
*** Settings ***
Documentation     This is the suite documentation:
...               Some: UTF-8 chars \ | € º ª ã É \

*** Test Cases ***
中文_is_chineseªºção_portuguese
    Log    The name of this test is "中文_is_chineseªºção_portuguese"
    Log To Console    This is at console: test is "中文_is_chineseªºção_portuguese"
```

Please test on your systems, and report back.

**Comment by ShaYouYu** [comment]
My screenshot of it installed in Python 3.8 . Help~
![1587893136(1)](https://user-images.githubusercontent.com/43426411/80303483-22d60a00-87e3-11ea-8f6e-ee3e7d474fe8.jpg)

**Comment by HelioGuilherme66** [comment]
Closing because should be fixed on next 2.0b1.
